.github/workflows/release.template.yml

```diff
@@ -115,6 +115,7 @@ jobs:
         env:
           INPUT_TAGS: ${{ steps.meta.outputs.tags }}
           INPUT_PUSH: true
+          INPUT_BUILD_ARGS: GIT_REF=${{ github.ref }}
```

Comment:
Yes, this will "store" the `GIT_REF` into the container environment variable. 
In the case of DSG, this will then be used by the DSG to display the version of the DSG in the build logs.

Reply:
https://docs.docker.com/engine/reference/commandline/build/#-set-build-time-variables---build-arg